SDL/Hit.cuh

```diff
@@ -66,60 +66,40 @@ namespace SDL
     CUDA_G void addHitToMemoryGPU(struct hits& hitsInCPU,struct modules& modulesInGPU,float x, float y, float z, unsigned int detId, unsigned int idxInNtuple,unsigned int moduleIndex, float phis,struct objectRanges& rangesInGPU);
     
     CUDA_HOSTDEV inline float ATan2(float y, float x) {
-    //if (x != 0) return  x * (float(-0xf.8eed2p-4) + x * x * float(0x3.1238p-4)); // degree 3 7 bit accuracy//atan2f(y, x);
-    if (x != 0){ 
-      //float a = y/x;
-      //float z = a*a;  
-      float test = atan2f(y, x);
-      //float test1 = a * (float(-0xf.8eed2p-4) + z * float(0x3.1238p-4)); 
-      //float test2 = a * (float(-0xf.ecfc8p-4) + z * (float(0x4.9e79dp-4) + z * float(-0x1.44f924p-4)));
-      //float test3 =  a * (float(-0xf.fcc7ap-4) + z * (float(0x5.23886p-4) + z * (float(-0x2.571968p-4) + z * float(0x9.fb05p-8))));
-      //float test4 = a * (float(-0xf.ff73ep-4) +
-      //        z * (float(0x5.48ee1p-4) +
-      //             z * (float(-0x2.e1efe8p-4) + z * (float(0x1.5cce54p-4) + z * float(-0x5.56245p-8)))));
-      //float test5 = a * (float(-0xf.ffff4p-4) +
-      //        z * (float(0x5.552f9p-4 + z * (float(-0x3.30f728p-4) +
-      //                                       z * (float(0x2.39826p-4) +
-      //                                            z * (float(-0x1.8a880cp-4) +
-      //                                                 z * (float(0xe.484d6p-8) +
-      //                                                      z * (float(-0x5.93d5p-8) + z * float(0x1.0875dcp-8)))))))));
-      //printf("%f %f %f %f %f %f %f %f\n",y,x,test,test1,test2,test3,test4,test5);
-      return test;
+      if (x != 0) return atan2f(y, x);
+      if (y == 0) return  0;
+      if (y >  0) return  float(M_PI) / 2.f;
+      else        return -float(M_PI) / 2.f;
```

Comment:
As mentioned in the context of the previous comment, in this PR I tried to only clean up some functions and not change their implementation. However, I agree that the part of code that you mention is needless. Checking the C++ `atan2f` implementation from `math.h`, it gives the same results as the behaviour we are trying to emulate here. Given this, I would just change `SDL::ATan2` to the standard `atan2f`. Maybe in another PR but it can be done in this one as well.

Reply:
sure ,can be in a follow up pr